### ISSUE TITLE
Installed Version does not work

### ISSUE THREAD

**Issue by braveheuel** [python-escpos/python-escpos#97]
I have checked out the latest version from the development branch (8b8162c1a943282bdcf9747ecfa6bff39af53e15) and did a "python3 setup.py install --user"
When I then try to load the module, I receive an error:

```
Python 3.4.3 (default, Jun 29 2015, 12:16:01) 
[GCC 5.1.1 20150618 (Red Hat 5.1.1-4)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import escpos.printer as printer
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/ch/mnt/raspberrypi/python-escpos/escpos/__init__.py", line 1, in <module>
    import constants
ImportError: No module named 'constants'
>>> import escpos
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/ch/mnt/raspberrypi/python-escpos/escpos/__init__.py", line 1, in <module>
    import constants
ImportError: No module named 'constants'
```

I can fix it with the attached patch. 
[escpos_import.txt](https://github.com/python-escpos/python-escpos/files/160453/escpos_import.txt)

I am unsure if that is the right way to do... The the import in the `__init__.py` even needed?

**Comment by patkan** [comment]
I think we do not need to import everything with the module. In #94 I have done it similar to your patch, but I am not sure if we even have to import `printer`. (I think it is simply for convenience)
Since `printer` is the only interface that is relevant to the user importing more makes no sense for me.

One small thing: I think you should cd somewhere else than your source directory. It seems to me that your python tries to importfrom there and not from the installed packages.

**Comment by braveheuel** [comment]
@patkan The error is always persistent, even cd to an other directory:

```
Python 3.4.2 (default, Oct 19 2014, 13:31:11) 
[GCC 4.9.1] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import escpos
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/ch/.local/lib/python3.4/site-packages/python_escpos-1.0.9_dev-py3.4.egg/escpos/__init__.py", line 1, in <module>
ImportError: No module named 'constants'
>>> 
```

**Comment by patkan** [comment]
@braveheuel Yeah this was just a remark on the side. You are right, the way the init.py imports the rest of the package might have been ok in Python 2 but definitely isn't in Python 3. The question is, whether we should completely remove the imports from init.py and regarding this I am unsure.

**Comment by braveheuel** [comment]
As far as I understand the documentation (https://docs.python.org/3.4/tutorial/modules.html#packages) the `__init__.py` only acts as initialization. We for now don't initialize something. The `__all__` magic word is needed for the "*" import.

So I think we are fine with removing all imports and keep the `__all__`.

Additionally: I should try out #97 as it supersedes this.

**Comment by patkan** [comment]
I just compared both behaviours:

#### with `from . import printer`
```
>>> import escpos
>>> p = escpos.printer.Serial("/dev/ttyUSB0")
Serial printer enabled

```
#### without ` from . import printer`
```
>>> import escpos
>>> p = escpos.printer.Serial("/dev/ttyUSB0")
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-2-0e0d19e40688> in <module>()
----> 1 p = escpos.printer.Serial("/dev/ttyUSB0")

AttributeError: 'module' object has no attribute 'printer'

```

So it depends on whether we want to give users the ability to use `import escpos` or whether they have to use `import escpos.printer`.
For me it would be ok to remove the import.

**Comment by braveheuel** [comment]
Ok, with the import line you don't have to explicitly import espos.printer, but you nevertheless need to use "escpos.printer.Serial" etc.

I am also ok with removing the line.

**Comment by patkan** [comment]
OK, I will commit a fix and then rebase the branch of my Pull request #94 onto this new commit.

**Comment by braveheuel** [comment]
As this is fixed in the PR #94 I will close this error.
